### PR TITLE
Restyled transaction view to match hifis

### LINKED ISSUE #1679: Restyle transaction webview to match hi-fi designs

### PR DESCRIPTION
**Pull request by erin-hughes**
Closes #1679 and contributes to #1155 

Signed-off-by: Erin Hughes <Erin.Hughes@ibm.com>

### CODE REVIEW

File: packages/blockchain-extension/extension/commands/openTransactionViewCommand.ts [hunk 1]
```diff
@@ -41,6 +42,9 @@ export async function openTransactionView(treeItem?: InstantiatedTreeItem): Prom
         }
     }
 
+    const gatewayRegistryEntry: FabricGatewayRegistryEntry = FabricGatewayConnectionManager.instance().getGatewayRegistryEntry();
+    const gatewayName: string = (gatewayRegistryEntry.name === 'local_fabric') ? 'Local Fabric' : gatewayRegistryEntry.name;
```
Comment (Jakeeyturner) [inline packages/blockchain-extension/extension/commands/openTransactionViewCommand.ts]: FabricRuntimeUtil.ts has constants for these these two!

File: packages/blockchain-extension/extension/commands/openTransactionViewCommand.ts [hunk 2]
```diff
@@ -41,6 +42,9 @@ export async function openTransactionView(treeItem?: InstantiatedTreeItem): Prom
         }
     }
 
+    const gatewayRegistryEntry: FabricGatewayRegistryEntry = FabricGatewayConnectionManager.instance().getGatewayRegistryEntry();
```
Comment (Jakeeyturner) [inline packages/blockchain-extension/extension/commands/openTransactionViewCommand.ts]: Shouldn't need to do this again

File: packages/blockchain-extension/extension/webview/TransactionView.ts [hunk 3]
```diff
@@ -15,22 +15,23 @@
 'use strict';
 import * as vscode from 'vscode';
 import { ReactView } from './ReactView';
-import { ExtensionCommands } from '../../ExtensionCommands';
+import { TransactionCreateView } from './TransactionCreateView';
 
 export class TransactionView extends ReactView {
-    public appState: any;
+    protected context: any;
```
Comment (Jakeeyturner) [inline packages/blockchain-extension/extension/webview/TransactionView.ts]: You shouldn't need to do nor set `this.context = context`, as context is a property on View (which ReactView extends).
You can just call this.context with no problem!

File: packages/blockchain-extension/test/commands/openTransactionViewCommand.test.ts [hunk 4]
```diff
@@ -153,6 +153,17 @@ describe('OpenTransactionViewCommand', () => {
             openViewStub.should.have.been.calledOnce;
         });
 
+        it('should correctly display the Local Fabric gateway name', async () => {
```
Comment (Jakeeyturner) [inline packages/blockchain-extension/test/commands/openTransactionViewCommand.test.ts]: Can use `FabricRuntimeUtil.LOCAL_FABRIC` and `FabricRuntimeUtil.LOCAL_FABRIC_DISPLAY_NAME` in the tests (and test name) as well.

**Comment by Jakeeyturner** [comment]
Other things to think about:
- Space between 'Smart contract' and the contract dropdown.
- Do we want icons on the panels?
- Do we want to be able to have multiple 'Create transaction page' tabs?
- Time Stamp should be Timestamp

File: packages/blockchain-extension/extension/commands/openTransactionViewCommand.ts (hunk 2 above)
Comment (erin-hughes) [reply packages/blockchain-extension/extension/commands/openTransactionViewCommand.ts]: Turns out I do need to do it this way 🙃

File: packages/blockchain-extension/extension/commands/openTransactionViewCommand.ts (hunk 1 above)
Comment (erin-hughes) [reply packages/blockchain-extension/extension/commands/openTransactionViewCommand.ts]: Fixed 👍

**Pull request merged**